main.cpp

```diff
@@ -415,7 +433,10 @@ struct Synthesizer
 {
     Piano piano;
     WaveGenerator oscillator;
-    WaveGenerator lfo;
+    float lfo;
+    float mParam1 = 10.0f;
+    float mParam2 = 13.3f;
+    float mParam3 = 3.7f;
```

Comment:
don't use `m` in member variable names
If a variable is used in a function, and it is not declared in that function or part of the function parameter list, it is obvious that the variable is a member variable. 
- [ ] revise